compiler/frontend/beam/src/main/java/edu/snu/nemo/compiler/frontend/beam/NemoPipelineVisitor.java

```diff
@@ -241,6 +241,19 @@ private static BeamCoder getCoderForView(final ViewFn viewFn, final Coder beamIn
    */
   private static DataCommunicationPatternProperty.Value getEdgeCommunicationPattern(final IRVertex src,
                                                                                     final IRVertex dst) {
+    try {
+      final Class<?> constructUnionTableFn = Class.forName(
+          "org.apache.beam.sdk.transforms.join.CoGroupByKey$ConstructUnionTableFn");
+      if (src instanceof OperatorVertex && ((OperatorVertex) src).getTransform() instanceof DoTransform
```

Comment:
Also for this bit of code, why is it that it cannot be included in the if-else statement below?